72.md

```diff
@@ -22,6 +24,7 @@ The goal of this NIP is to create moderator-approved public communities around a
     ["d", "<Community name>"],
     ["description", "<Community description>"],
     ["image", "<Community image url>", "<Width>x<Height>"],
+    ["access", "open|closed|hybrid"],
 
```

Comment:
Shouldn't the current sharedkey's public key be listed as a reference to decrypt the group? Maybe an `e` tag to the most recent `kind:24` for this group? 

Clients then look at the public key/e-tag and check if they have a private key in a `kind:24` that matches this public key.

Reply:
Just pushed an update to nip 87 that specifies that the key share event should include an `a` tag referencing the group. Anyone who has the key will know about it.